src/env.rs

```diff
@@ -310,15 +314,16 @@ where
         self.env.set_map_size(size).map_err(Into::into)
     }
 
-    /// Closes this environment and deletes all its files from disk. Doesn't delete the
-    /// folder used when opening the environment.
-    pub fn close_and_delete(self) -> Result<(), StoreError> {
+    /// Closes this environment and optionally deletes all its files from disk. Doesn't
+    /// delete the folder used when opening the environment.
+    pub fn close(self, options: CloseOptions) -> Result<(), CloseError> {
         let files = self.env.get_files_on_disk();
-        self.sync(true)?;
```

Comment:
Why the force sync is no longer needed here for both backends? If it's still useful to the consumers, we can add it to `CloseOptions` as another option.

Reply:
It was never needed to begin with! I initially added it because it seemed like a good default, but we never open LMDB with MDB_NOSYNC, and in the case of safe-mode it's a noop.